detox/src/utils/environment.js

```diff
@@ -171,24 +172,36 @@ function throwMissingGmsaasError() {
   throw new DetoxRuntimeError(`Failed to locate Genymotion's gmsaas executable. Please add it to your $PATH variable!\nPATH is currently set to: ${process.env.PATH}`);
 }
 
-function getDetoxVersion() {
+const getDetoxVersion = _.memoize(() => {
```

Comment:
Rather a nitpicking, but you don't need to use `memoize` here. You'll be absolutely fine with `_.once` since your functions don't require any arguments. It is a more lightweight implementation, and its mental model is a bit easier/lighter.

Reply:
oh, great